pkg/client/camel/applyconfiguration/camel/v1/integrationplatformbuildspec.go

```diff
@@ -27,15 +27,16 @@ import (
 // IntegrationPlatformBuildSpecApplyConfiguration represents an declarative configuration of the IntegrationPlatformBuildSpec type for use
 // with apply.
 type IntegrationPlatformBuildSpecApplyConfiguration struct {
-	BuildStrategy          *v1.BuildStrategy                           `json:"buildStrategy,omitempty"`
-	PublishStrategy        *v1.IntegrationPlatformBuildPublishStrategy `json:"publishStrategy,omitempty"`
-	RuntimeVersion         *string                                     `json:"runtimeVersion,omitempty"`
-	RuntimeProvider        *v1.RuntimeProvider                         `json:"runtimeProvider,omitempty"`
-	BaseImage              *string                                     `json:"baseImage,omitempty"`
-	Registry               *RegistrySpecApplyConfiguration             `json:"registry,omitempty"`
-	Timeout                *metav1.Duration                            `json:"timeout,omitempty"`
-	Maven                  *MavenSpecApplyConfiguration                `json:"maven,omitempty"`
-	PublishStrategyOptions map[string]string                           `json:"PublishStrategyOptions,omitempty"`
+	BuildStrategy           *v1.BuildStrategy                           `json:"buildStrategy,omitempty"`
+	PublishStrategy         *v1.IntegrationPlatformBuildPublishStrategy `json:"publishStrategy,omitempty"`
+	RuntimeVersion          *string                                     `json:"runtimeVersion,omitempty"`
+	RuntimeProvider         *v1.RuntimeProvider                         `json:"runtimeProvider,omitempty"`
+	BaseImage               *string                                     `json:"baseImage,omitempty"`
+	Registry                *RegistrySpecApplyConfiguration             `json:"registry,omitempty"`
+	BuildCatalogToolTimeout *int                                        `json:"buildCatalogToolTimeout,omitempty"`
```

Comment:
Just curious : why not use the same type as the `Timeout` parameter ?

Reply:
Good point. I will change accordingly.